src/it/scala/temple/EndpointIntegrationTest.scala

```diff
@@ -11,4 +13,11 @@ class EndpointIntegrationTest extends EndpointTesterSpec with Matchers {
     val simpleTemple = FileUtils.readFile("src/test/scala/temple/testfiles/simple-dc.temple")
     noException should be thrownBy buildAndTestEndpoints(simpleTemple)
   }
+
+  it should "correctly generate the example Templefiles" in {
+    Files.list(Paths.get("examples")).map(path => path.toString).forEach { (path: String) =>
```

Comment:
Worth filtering for `*.temple` so we can add a readme to that folder without it being imported?